wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -903,4 +921,124 @@ public static function setIndianStateOptions(string $entity, string $field, arra
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function mailNotificationToMMT($op, $groupID, $entityID, &$params) {
+    if ($op !== 'create') {
+      return;
+    }
+
+    if (!($goonjField = self::findOfficeId($params))) {
+      return;
+    }
+
+    $goonjFieldId = $goonjField['value'];
+    $vehicleDispatcheId = $goonjField['entity_id'];
+
+    $collectionSourceVehicleDispatche = EckEntity::get('Collection_Source_Vehicle_Dispatch', FALSE)
+      ->addSelect('Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id')
+      ->addWhere('id', '=', $vehicleDispatcheId)
+      ->execute()->first();
+
+    $collectionCampId = $collectionSourceVehicleDispatche['Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id'];
+
+    $coordinators = Relationship::get(FALSE)
+      ->addWhere('contact_id_b', '=', $goonjFieldId)
+      ->addWhere('relationship_type_id:name', '=', self::MATERIAL_RELATIONSHIP_TYPE_NAME)
+      ->addWhere('is_current', '=', TRUE)
+      ->execute()->first();
+
+    $mmtId = $coordinators['contact_id_a'];
+
+    if (empty($mmtId)) {
+      return;
+    }
+
+    $email = Email::get(FALSE)
+      ->addSelect('email', 'contact_id.display_name')
+      ->addWhere('contact_id', '=', $mmtId)
+      ->execute()->single();
+
+    $mmtEmail = $email['email'];
+    $contactName = $email['contact_id.display_name'];
+
+    // Email to material management team member.
+    $mailParams = [
+      'subject' => 'New Entry For Matrial Dispatch Notification',
+      'from' => 'urban.ops@goonj.org',
```

Comment:
Is this required?

Reply:
Yes we are sending email to mmt  from here

Reply:
In CiviCRM we can go to `Administer > CiviMail > From Address` and add multiple from address. So instead of hardcoding them here, we can fetch it from there. I've already done this here where it fetches the default one: https://github.com/ColoredCow/goonj/blob/eb968c52f445da9083001ea57ab16c114c53ecfe/wp-content/civi-extensions/goonjcustom/Civi/CollectionBaseService.php#L319-L323